docs/ui/auth/fragments/web/authenticator.md

```diff
@@ -868,6 +869,67 @@ export withAuthenticator(App, {initialAuthState: 'signup'});
 ```
 </docs-filter>
 
+## Methods & Enums
+
+### AuthState
+
+`AuthState` is an enum with the following values:
+
+```js
+enum AuthState {
```

Comment:
Would it be beneficial to document _when_ each one of these is achieved, and how each one transitions to the others?